src/controllers/employee.controller.ts

```diff
@@ -7,21 +7,22 @@ const employeeController = async (request: Request, h: ResponseToolkit): Promise
   switch (request.method) {
     case 'get': {
       const allEmployees = await allEmployeesDetails().then((results) => results);
-      return h.response(response(!allEmployees.length ? 500 : 200, allEmployees));
+      return h.response(response(!allEmployees.length ? 500 : 200, allEmployees))
+        .code(!allEmployees.length ? 500 : 200);
```

Comment:
why do we require this `.code(!allEmployees.length ? 500 : 200);` for every response